android/lib/build.gradle

```diff
@@ -1,19 +1,15 @@
-apply plugin: 'com.android.library'
-
-repositories {
-    maven { url 'https://repo1.maven.org/maven2' }
+plugins {
+    id "com.android.library"
 }
 
 android {
-    compileSdkVersion 23
-    buildToolsVersion '25.0.0'
+    compileSdkVersion 30
 
     defaultConfig {
-        minSdkVersion 16
-        targetSdkVersion 23
-        versionCode 1
-        versionName "1.0"
+        minSdkVersion 21
+        targetSdkVersion 30
```

Comment:
Out of curiosity, is there a way to inherit these settings from `android/build.gradle`?

Reply:
We could add a property to the extension, but I think this one needs a little bit of a project structure re-work. I don't think that change would be high impact, so I am leaving it as is.

Reply:
Sure. Thanks for the explanation 😄